audio/pcaudiolib/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                pcaudiolib
+github.setup        espeak-ng pcaudiolib 1.1
+version             1.1
+revision            1
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         Portable C Audio Library
+
+long_description    Portable C Audio Library
+
+homepage            https://github.com/espeak-ng/pcaudiolib
+distname            ${version}
+checksums           rmd160  f68876d3346734e993b9d5ebfc3ef48de9161851 \
+                    sha256  449d69c3fd4ed0136d357fd42a73accb276da9b35a19258a379e3ac7ac88c148 \
+                    size    34082
```

Comment:
Please use `github.tarball_from archive`. This will require changing the checksums as well.